### PR TITLE
2015 10 cc new report ui

### PR DESCRIPTION
**Pull request by local-girl**
Added new abuse overlay as per Nadines requests. There is a new flag icon and color. Also the overlay now has help text with an optional link to nettiquitte url (only used in Mercator for now)

### CODE REVIEW

**Comment by MagdaN** [comment]
I get a sass errror

**Comment by local-girl** [comment]
I think this is in master

**Comment by local-girl** [comment]
I fixed it in this branch by adding the @optional like it says but I never discovered what the underlying problem was

**Comment by local-girl** [comment]
This seems to have been fixed upstream

**Comment by MagdaN** [comment]
works

File: src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Comment/Column.html [hunk 1]
```diff
@@ -1,8 +1,14 @@
 <div data-ng-switch="transclusionId">
     <div data-ng-switch-when="menu">
-        <div class="moving-column-tab">
-            <i class="icon-speechbubbles moving-column-icon"></i>
-            {{ "TR__COMMENTS" | translate }}
+        <div class="moving-column-tab" data-ng-switch on="overlay">
+            <span ng-switch-when="abuse">
```
Comment (xi) [inline src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Comment/Column.html]: `data-` prefix missing

**Pull request merged**